.github/ISSUE_TEMPLATE.md

```diff
@@ -10,3 +10,5 @@ Welllllll, it just might be.  Do me a favor and check the following.
 
 Also, for the quickest assistance please run `ignite doctor` and paste in
 everything (including the lines with 3 backticks).
```

Comment:
I did ya in Ignite 2.0.  I was thinking about a `ignite issue` command to launch a browser with the template in there.